__tests__/atomWithLocation_spec.tsx

```diff
@@ -114,6 +114,75 @@ describe('atomWithLocation', () => {
     expect(window.location.pathname).toEqual('/1');
     expect(window.history.length).toEqual(3);
   });
+
+  it('can override atomOptions', async () => {
+    const locationAtom = atomWithLocation({ replace: false });
+
+    const Navigation = () => {
+      const [location, setLocation] = useAtom(locationAtom);
+      return (
+        <>
+          <div> current pathname in atomWithLocation: {location.pathname} </div>
+          <button type="button" onClick={() => window.history.back()}>
+            back
+          </button>
+          <button type="button" onClick={() => setLocation({ pathname: '/1' })}>
+            button1
+          </button>
+          <button
+            type="button"
+            onClick={() =>
+              setLocation(
+                {
+                  pathname: '/2',
+                },
+                // @ts-expect-error: TODO: How can we fix this error? Expected 1 arguments, but got 2
+                { replace: true },
```

Comment:
Not sure how to update type to allow this to be passed?

Reply:
`PrimitiveAtom<Location>;` and `PrimitiveAtom<T>;` should be something like this:
```ts
type Option = { replace: boolean };

WritableAtom<Location, [SetStateAction<Location>, Option?], void>
```